### PR TITLE
Fix error when we do request to status of coverage without user informations.

### PR DESCRIPTION
**Pull request by Netmisa**
# Description

This PR fix error when we do request to status of coverage without user authentification

## How to test

This message should work

```
{"coverages":[{"region_id":"fr-cen"}],"request_duration":4,"application_id":-1,"request_date":1505329195,"token":"XXXXX","client":"XXXXXX","host":"http://api.navitia.io/","api":"v1.status","response_size":280,"application_name":"","path":"/v1/coverage/fr-cen/status"}
```

### CODE REVIEW

File: analyzers/requests_calls.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def collect_data(self, dataframe):
             when(dataframe.coverages[0].region_id.isNull(), '').
             otherwise(dataframe.coverages[0].region_id).alias('region_id'),
             dataframe.api,
-            dataframe.user_id,
+            when(dataframe.user_id.isNull(), 0).otherwise(dataframe.user_id).alias('user_id'),
```
Comment (nberard) [inline analyzers/requests_calls.py]: can u add a test for that case? =D

Comment (nberard) [inline analyzers/requests_calls.py]: shouldn't we handle case in users as well?

Comment (Netmisa) [reply analyzers/requests_calls.py]: Of course ! :)

File: analyzers/users.py [hunk 2]
```diff
@@ -14,9 +14,9 @@ def collect_data(self, dataframe):
 
             new_users = dataframe \
                 .select(
-                    "user_id",
-                    first('user_name').over(wdesc).alias('user_name'),
-                    first('request_date').over(wasc).alias('first_date')
+                    when(dataframe.user_id.isNull(), 0).otherwise(dataframe.user_id).alias('user_id'),
+                    when(dataframe.user_name.isNull(), 'unknown').otherwise(first(dataframe.user_name).over(wdesc)).alias('user_name'),
+                    first(dataframe.request_date).over(wasc).alias('first_date')
```
Comment (nberard) [inline analyzers/users.py]: there should be a modif in insert_or_update as well, maybe it's why tests are failing

**Pull request merged**